frontend/frontend/src/components/Profile.vue

```diff
@@ -16,6 +16,7 @@
 				v-for="artist in artists"
 				:artistName="artist.name"
 				:artistSummary="artist.summary"
+				:maxTime = "this.$route.query.selectDifficulty"
```

Comment:
Hi! I think this line of code may be causing the profile page to not display. The "this" object is not accessible from the table and therefore this.$route.query.selectDifficulty is undefined. Based on my test all I think we have to change is adding a maxTime variable in the data(){} section that is equal to this.$route.query.selectDifficulty (where  the "this" object is accessible from. and then changing :maxTime in the <Artists /> prop to be :maxTime="maxTime". Let me know if you want to talk  more about this!